osv/logging_helper.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 Google LLC
```

Comment:
nit: can we just name this module "logging" ? "helper" is generally not a very descriptive word for naming.

Reply:
I could do that, but I'm a bit wary of some ambiguity - osv's `__init__.py` imports everything from a few files, including the python `logging` module. So you end up with something like
```python
>>> import osv
>>> osv.logging
"<module 'logging' from '/usr/lib/python3.11/logging/__init__.py'>"
>>> import osv.logging
>>> osv.logging
"<module 'osv.logging' from '/path/to/osv.dev/osv/logging.py'>"
```

(I actually forgot to import `osv.logging_helper` anyway - will fix that)

Reply:
Ah, very good point here. 

In that case, can we name this something like "logs" instead?

Reply:
Changed to `osv.logs`